addons/binding/org.openhab.binding.jeelink/ESH-INF/i18n/jeelink_de.properties

```diff
@@ -0,0 +1,63 @@
+# binding
+binding.jeelink.name = JeeLink Binding
+binding.jeelink.description = Binding f\u00fcr JeeLink USB Empf\u00e4nger und EC3000 Energiekosten-Messger\u00e4te / Lacrosse Temperatur-Sensoren.
```

Comment:
I had Umlauts in there a while ago, but that did not work at the time. Has this changed or did I do something else wrong? I have changed this now to using Umlauts and it works.